server/models/house.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+const slug = require('slug');
+mongoose.Promise = global.Promise;
+
+const houseSchema = new Schema({
+  title: {
+    type: String,
+    trim: true,
+    required: 'Title is required'
+  },
+  slug: String,
+  description: {
+    type: String,
+    trim: true
+  },
+  created: {
+    type: Date,
+    default: Date.now
+  },
+  location: {
+    postCode: {
+      type: String,
+      required: 'Postcode is required'
+    },
+    street: {
+      type: String,
+      required: 'Street is required'
```

Comment:
Why do we need location and street, can we not have a schema that defines an address with Streetname POSTCode and county rather than having address rely on two different parts of the schema ?